src/openeo_test_suite/tests/workflows/test_add_dimension.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
+
+import xarray as xr
+
+
+def test_ndvi_add_dim(cube_one_day_red_nir, b_dim, tmp_path):
+    filename = tmp_path / "test_ndvi_add_dim.nc"
+
+    def compute_ndvi(data):
+        from openeo.processes import array_element
+
+        red = data.array_element(index=0)
+        nir = data.array_element(index=1)
+        return (nir - red) / (nir + red)
+
+    ndvi = cube_one_day_red_nir.reduce_dimension(dimension=b_dim, reducer=compute_ndvi)
+    ndvi = ndvi.add_dimension(type="bands", name=b_dim, label="NDVI")
+    ndvi.download(filename)
+
+    assert Path(filename).exists()
```

Comment:
FYI: this should work as `filename` is already a `Path`
```suggestion
    assert filename.exists()
```